drag_and_drop_v2/templates/html/drag_and_drop.html

```diff
@@ -1,4 +1,4 @@
 {% load i18n %}
 <section class="xblock--drag-and-drop">
-    {% trans "Loading drag and drop exercise." %}
+    <i class="fa fa-spin fa-spinner"></i>{% trans "Loading drag and drop exercise." %}
```

Comment:
@e-kolpakov There is no/very little horizontal space between the spinner and the message. Can you make the right-hand margin of the spinner shown in this context consistent with the right-hand margin of the spinner that is shown when dropping an item?